.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: java
```

Comment:
I don't get it, why is it so easy to do this on Travis and so hard for our users to install of the software?